x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/criteria_conditions.tsx

```diff
@@ -67,11 +81,30 @@ export const CriteriaConditions = memo<CriteriaConditionsProps>(
         .join(', ');
     }, [os]);
 
+    const getEntryValue = useCallback((type: string, value: string | string[]) => {
```

Comment:
nit: `useCallback` without dependencies is less performant than not using `useCallback`